dynamic/variables.tf

```
@@ -21,6 +21,18 @@ variable "aws_region" {
   default     = "us-east-1"
 }
 
+variable "ec2readonly_role_description" {
+  type        = string
+  description = "The description to associate with the IAM role (as well as the corresponding policy) that allows read access to some EC2 attributes in the dynamic account."
+  default     = "Allows read access to some EC2 attributes in the dynamic account."
+}
+
+variable "ec2readonly_role_name" {
+  type        = string
+  description = "The name to assign the IAM role (as well as the corresponding policy) that read access to some EC2 attributes in the dynamic account."
```

Just to make things easier with the other suggestion.

```suggestion
  description = "The name to assign the IAM role (as well as the corresponding policy) that allows read access to some EC2 attributes in the dynamic account."
```

---

Thanks! Resolved with https://github.com/cisagov/cool-accounts/pull/126/commits/41ba21342024429e905c8327ffec2b168bf1da5c.